archunit/src/jdk14test/java/com/tngtech/archunit/core/importer/ClassFileImporterRecordsTest.java

```diff
@@ -0,0 +1,59 @@
+package com.tngtech.archunit.core.importer;
+
+import com.tngtech.archunit.core.domain.JavaClass;
+import com.tngtech.archunit.core.domain.JavaConstructor;
+import com.tngtech.archunit.core.domain.JavaField;
+import com.tngtech.archunit.core.domain.JavaMethod;
+import org.junit.Test;
+
+import static com.tngtech.archunit.testutil.Assertions.assertThat;
+
+public class ClassFileImporterRecordsTest {
+
+    @Test
+    public void imports_simple_record() {
+        record RecordToImport(String component1, int component2) {
+        }
+
+        JavaClass javaClass = new ClassFileImporter().importClasses(RecordToImport.class, Record.class).get(RecordToImport.class);
+
+        assertThat(javaClass)
+                .matches(RecordToImport.class)
+                .hasRawSuperclassMatching(Record.class)
+                .hasNoInterfaces()
+                .isInterface(false)
+                .isEnum(false)
+                .isAnnotation(false)
+                .isRecord(true);
+    }
+
+    @Test
+    public void imports_constructor() throws Exception {
+        record RecordToImport(String component1, int component2) {
+        }
+
+        JavaClass javaClass = new ClassFileImporter().importClass(RecordToImport.class);
+        JavaConstructor constructor = javaClass.getConstructor(String.class, int.class);
+        assertThat(constructor).isEquivalentTo(RecordToImport.class.getDeclaredConstructor(String.class, int.class));
+    }
+
+    @Test
+    public void imports_record_component_fields() throws Exception {
+        record RecordToImport(String component1, int component2) {
+        }
+
+        JavaClass javaClass = new ClassFileImporter().importClass(RecordToImport.class);
+        JavaField component1Field = javaClass.getField("component1");
+        assertThat(component1Field).isEquivalentTo(RecordToImport.class.getDeclaredField("component1"));
+    }
+
+    @Test
+    public void imports_record_component_getters() throws Exception {
+        record RecordToImport(String component1, int component2) {
+        }
+
+        JavaClass javaClass = new ClassFileImporter().importClass(RecordToImport.class);
+        JavaMethod component1Method = javaClass.getMethod("component1");
+        assertThat(component1Method).isEquivalentTo(RecordToImport.class.getMethod("component1"));
```

Comment:
Given that we use `getDeclaredConstructor` and `getDeclaredField` above: should we use `getDeclaredMethod` here as well?